.cirrus.yml

```diff
@@ -41,6 +40,11 @@ global_task_template: &GLOBAL_TASK_TEMPLATE
   ci_script:
     - ./ci/test_run_all.sh
 
+compute_credits_template: &CREDITS_TEMPLATE
+  # https://cirrus-ci.org/pricing/#compute-credits
+  # Only use credits for pull requests to the main repo
+  use_compute_credits: $CIRRUS_REPO_FULL_NAME == 'bitcoin/bitcoin' && $CIRRUS_PR != ""
```

Comment:
Are pulls into the GUI repo skipped intentionally?

Reply:
Yes, it is a separate org, so billing would need to be set up separately. Is there a lot of traffic on the gui repo, so that this is needed?

Reply:
No much traffic :)